lib/scrabble.rb

```diff
@@ -1,4 +1,63 @@
 module Scrabble
   # require your gems and classes here
   # require_relative 'scrabble/whatevs'
+  class Scrabble
+
+    def self.score(word)
+      value = { 
+                 1 => ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"],
+                 2 => ["d", "g"], 
+                 3 => ["b", "c", "m", "p"],
+                 4 => ["f", "h", "v", "w", "y"],
+                 5 => ["k"],
+                 8 => ["j", "x"],
+                 10 => ["q", "z"]
+                }
+
+      count = 0
+        word_array = word.to_s.downcase.split(//)
+        return "too many letters" if word_array.length > 7 
+        return "error" if word.to_s.strip.empty?
+        word_array.each do |l|
```

Comment:
I would prefer if you used a more descriptive variable name like `letter`, `key` and `value`